content/css/concepts/animations/terms/background-position/background-position.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'background-position'
```

Comment:
This is incorrect. This file already exists - https://www.codecademy.com/resources/docs/css/background/background-position

The PR is for the addition of images that match the examples in the existing entry. For example,
https://www.codecademy.com/resources/docs/css/animations/animation

Reply:
can you tell me in which file the images needs to be added? The issue raised said that this file did not existed so I created one.

Reply:
There should be images added that represent the examples in the entry